packages/@okta/vuepress-site/guides/sign-into-mobile-app/get-user-info/android/getinfo.md

```diff
@@ -1 +1,17 @@
-retrieve user info code example
\ No newline at end of file
+An example of getting user information from userinfo endpoint:
```

Comment:
The following is an example of getting user information from the `/userinfo` endpoint: